docs/developer/add-new-source.md

```diff
@@ -1,6 +1,14 @@
 # How to add a new source to the Mondo Ingest
 
-1. Add a new import and component to [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
-2. Add a new metadata file to [src/ontology/metadata](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata). It is important we try and document as much about the source as we can.
-3. Update the Website Table of Contents [mkdocs.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/mkdocs.yaml)
+## 1. `mondo-ingest-odk.yaml`
+### 1.1. Add source info
+Add a new filename and URL to the `products` > `components` section in [src/ontology/mondo-ingest-odk.yaml](https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/mondo-ingest-odk.yaml).
```

Comment:
@matentzn Can you confirm that what I've written here looks like what we want? This previously said "add new import and component". The `imports` section of `mondo-ingest-odk.yaml` has RO and OMO in it, whereas the `products.components` section has what I'm more familiar with.

Reply:
component is correct, so what you have now is right